### ISSUE TITLE
ytplayer id

### ISSUE THREAD

**Issue by Alex-Jordan** [rbeezer/mathbook#395]
At line 3940 of `mathbook-html.xsl` there is 

```
    <xsl:element name="iframe">
        <xsl:attribute name="id">ytplayer</xsl:attribute>
```

When multiple youtube videos are in a document, this can make a single page have several iframes with the same id. I think this leads to error messages from `xsltproc` that I get, like:

```
element iframe: validity error : ID ytplayer already defined
element iframe: validity error : ID ytplayer already defined
element iframe: validity error : ID ytplayer already defined
```
I even get this from the sample article:

```
syst203m4138:sample-article alex.jordan$ xsltproc ../../xsl/mathbook-html.xsl sample-article.xml 
element iframe: validity error : ID ytplayer already defined
element iframe: validity error : ID ytplayer already defined
```

**Comment by rbeezer** [comment]
Interesting, I don't get this error message (which is about the generated output, not the XML source).  
@twjudson  is also getting errors/messages that I don't receive (with content I don't understand).

Anyway, this should be fixed at a1908496  

When  you get a chance, can you verify that the error messages go away?  If so, you can close this as well.

Everything in Chapter 15 of the sample article seems fine, except that my Led Zeppelin video got taken down for embedding.

**Comment by Alex-Jordan** [comment]
No errors now. From the sample article, I get this error:

```
element section: validity error : ID  already defined
```

(which I do believe I was getting before). It seems like it could be the same kind of thing. Here is my output form `xsltproc --version`. Does it match yours?

```
Using libxml 20902, libxslt 10128 and libexslt 817
xsltproc was compiled against libxml 20902, libxslt 10128 and libexslt 817
libxslt 10128 was compiled against libxml 20902
libexslt 817 was compiled against libxml 20902
```

**Comment by rbeezer** [comment]
I did this with Tom.  Only one discrepancy in versioning.

That's the error Tom gets - note, no ID is shown like before.  AATA spits out about 23 of these, I always assumed it was source, but maybe not?  I got it before, but not now.

If you track this one down, that'd be great.  But I'm not going to sweat it - I think it is harmless.  I'm going to close this, open something new if you want about that one "new" error.

```
rob@lava:~$ xsltproc --version
Using libxml 20903, libxslt 10128 and libexslt 817
xsltproc was compiled against libxml 20902, libxslt 10128 and libexslt 817
libxslt 10128 was compiled against libxml 20902
libexslt 817 was compiled against libxml 20902
```
